internal/command/testdata/init-get/output.jsonlog

```diff
@@ -0,0 +1,8 @@
+{"@level":"info","@message":"Terraform 1.9.0-dev","@module":"terraform.ui","terraform":"1.9.0-dev","type":"version","ui":"1.2"}
+{"@level":"info","@message":"Initializing the backend...","@module":"terraform.ui","type":"init_output"}
+{"@level":"info","@message":"Initializing modules...","@module":"terraform.ui","type":"init_output"}
+{"@level":"info","@message":"- foo in foo","@module":"terraform.ui","type":"log"}
```

Comment:
It would be great to have more useful semantic output for module and provider installations, so that clients could build useful UI on top of it. Not necessary for this PR, but if you can sketch out how that might work in the PR description it'd help make sure we're not painting ourselves into a corner.